Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp

```diff
@@ -447,8 +479,9 @@ namespace AzFramework
             {
                 if (entity != nullptr)
                 {
+                    entity->SetSpawnTicketId(0);
```

Comment:
Nitpick: Would put a comment here to explain that without resetting the id here there would be an infinite loop of destroy calls.